python/lsst/ip/isr/isrTask.py

```diff
@@ -846,12 +845,12 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             # However, these are packed oddly, so replace the data
             # reference with a URI that can be handled by the
             # straylight correction.  This breaks IO purity.
-            straylightRef = inputs['strayLightData']
-            diskUri = straylightRef.getUri()
-            inputs['strayLightData'] = diskUri
+            # straylightRef = inputs['strayLightData']
+            # diskUri = straylightRef.getUri()
+            # inputs['strayLightData'] = diskUri
+            print("Recieved straylightData: ", inputs['strayLightData'])
```

Comment:
Probably remove this debug print and put back in the comments?

Reply:
I'm rewriting this, as butler now handles StrayLightData objects.  This mitigates the need for the getUri, as inputs['strayLightData'] now contains the object itself.